dashboard/test/controllers/api/v1/sections_students_controller_test.rb

```diff
@@ -71,4 +71,49 @@ class Api::V1::SectionsStudentsControllerTest < ActionController::TestCase
     put :update, params: {section_id: @section.id, id: @student.id, student: {gender: 'f'}}
     assert_response :forbidden
   end
+
+  test 'teacher can add one student to a word section' do
+    sign_in @teacher
+    post :bulk_add, params: {section_id: @section.id, students: [{gender: 'f', age: 9, name: 'name'}]}
+    assert_response :success
+
+    parsed_response = JSON.parse(@response.body)
+    assert_equal 'name', parsed_response[0]['name']
+    assert_equal 9, parsed_response[0]['age']
+    assert_equal 'f', parsed_response[0]['gender']
+
+    new_student = User.find_by_id(parsed_response[0]['id'])
+
+    assert_equal 'name', new_student.name
+    assert_equal 9, new_student.age
+    assert_equal 'f', new_student.gender
+  end
+
+  test 'teacher can add multiple student to a word section' do
+    sign_in @teacher
+    post :bulk_add, params: {section_id: @section.id,
+      students: [{gender: 'f', age: 10, name: 'name1'}, {gender: 'm', age: 10, name: 'name2'}]}
+    assert_response :success
+    assert_equal 2, JSON.parse(@response.body).length
```

Comment:
suggestion: to ensure it actually created the students too, you can wrap the action on line 94 with 
```ruby
assert_creates Student, 2 do 
```

and similarly on the bad_request tests below, wrap in 
```ruby
assert_does_not_create Student do
```